portal-impl/test/unit/com/liferay/portal/util/PortalImplEscapeRedirectTest.java

```diff
@@ -128,6 +173,17 @@ public void testEscapeRedirectWithDomains() throws Exception {
 		Assert.assertNull(_portalImpl.escapeRedirect("http:web"));
 		Assert.assertNull(
 			_portalImpl.escapeRedirect("https://google.com\uFFFD@localhost"));
+
+		// Portal domains
```

Comment:
changed to a single comment section to represent the use case of `PORTAL_DOMAINS`